content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

```diff
@@ -0,0 +1,33 @@
+---
+title: kubeadm reset
+content_type: conceito
+weight: 60
+---
+<!-- overview -->
+Executa uma melhor reversão das alterações feitas pelo `kubeadm init` ou `kubeadm join`.
```

Comment:
You used "executa o melhor esforço para reverter" in other places. I think it fits very well here too.